hugo/content/045_contribute/20_documentation/25_markup/notice.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Notice
+description : "Disclaimers to help you structure your page"
+---
+
+The notice shortcode shows 4 types of disclaimers to help you structure your page.
+
+### Note
+
+```
+{{%/* notice note */%}}
+A notice disclaimer
```

Comment:
* all these specially rendered notices are rendered using a very small font
* this page uses too many colors, maybe use a light theme for the code snippets to get rid of the black boxes

Reply:
"too many colors" ...

this is a sample page which show which styles is possible. It is up to the document developer to use it in a real document or style the page in a clean way. But in an sample page we should show what is possible.

font is adapted (incremented)